src/PKSim.Core/Services/SimulationBuildingBlockUpdater.cs

```diff
@@ -59,6 +59,11 @@ public interface ISimulationBuildingBlockUpdater
       void UpdateFormulationsInSimulation(Simulation simulation);
 
       bool BuildingBlockSupportsQuickUpdate(IPKSimBuildingBlock templateBuildingBlock);
+
+      /// <summary>
+      /// Returns whether a building block comparison is available for the building block
+      /// </summary>
+      bool BuildingBlockSupportComparison(IPKSimBuildingBlock templateBuildingBlock);
```

Comment:
In our case, we can compare BB even though not quick update is possible. The assumption is not valid anymore